contactos/tests/contacts_tests.py

```diff
@@ -228,7 +228,7 @@ def test_it_sets_the_outbound_message_to_ready(self):
 class ResendOutboundMessages(TestCase):
     def setUp(self):
         super(ResendOutboundMessages, self).setUp()
-
+        settings.FLAG_BOUNCED_CONTACTS = True
```

Comment:
Please modify the settings temporarily per test case that needs it, otherwise we're potentially making other tests invalid

https://docs.djangoproject.com/en/1.8/topics/testing/tools/#overriding-settings